### PR TITLE
Update svg: paint-order support in firefox

### PR DESCRIPTION
**Pull request by smalllong**
#### Summary

Update svg: paint-order support in firefox
reference: https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/21#svg

### CODE REVIEW

File: svg/attributes/paint-order.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
               "version_added": "≤79"
             },
             "firefox": {
-              "version_added": true
+              "version_added": "21"
             },
             "firefox_android": {
               "version_added": true
```
Comment (queengooborg) [inline svg/attributes/paint-order.json:22]:
```suggestion
              "version_added": "21"
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Nice find, thanks!  May we also set Firefox Android to the same version number?

File: svg/attributes/paint-order.json (hunk 1 above)
Comment (smalllong) [reply svg/attributes/paint-order.json:22]: I don't know about Android version so please leave it unchanged

**Review by ddbeck — APPROVED** [review]
I can't find any evidence that Android was unsupported (Firefox desktop and Android are typically the same). I'm going to apply the suggested change and merge it. Thank you, @smalllong! 🎉

**Pull request merged**